unison-cli/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -358,7 +358,14 @@ run verbosity dir stanzas codebase runtime sbRuntime nRuntime config ucmVersion
                         let getRoot = fmap Branch.head . atomically $ readTMVar rootVar
                         liftIO (parseInput codebase getRoot curPath numberedArgs patternMap args) >>= \case
                           -- invalid command is treated as a failure
-                          Left msg -> liftIO (dieWithMsg $ Pretty.toPlain terminalWidth msg)
+                          Left msg -> do
```

Comment:
Previously this would crash the transcript even if inside a `ucm:error` block, now it prints the error the output but proceeds if it was an expected error.